lessons/specifics/poolboy.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+title: Poolboy
+category: specifics
+order: 1
```

Comment:
Yes, this section idea looks promising. Though, it may become opinionated after some time.